dashboard/test/models/section_test.rb

```diff
@@ -10,6 +10,49 @@ class SectionTest < ActiveSupport::TestCase
     @default_attrs = {user: @teacher, name: 'test-section'}
   end
 
+  test "sections are soft-deleted" do
+    assert_no_change("Section.with_deleted.count") do
+      @section.destroy
+      assert @section.reload.deleted?
+    end
+  end
+
+  test "destroying section destroys appropriate followers" do
+    already_deleted_follower = nil
+    Timecop.travel DateTime.new(2017, 2, 18, 12, 0, 0) do
```

Comment:
The `deletion_time` variable in my example is that timestamp. You could use it in the assertion below. I didn't make this clear, but that's why I separated it into 2 lines.